apps/playwright-browser-tunnel/tests/testFixture.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { test as base } from '@playwright/test';
+import { tunneledBrowser } from '../src/tunneledBrowserConnection';
+
+export const test = base.extend({
+  browser: [
+    async ({ browserName, launchOptions, channel, headless }, use) => {
+      console.log(`Starting tunnel server for browser: ${browserName}, channel: ${channel}`);
+
+      await using tunnel = await tunneledBrowser(browserName, {
+        channel,
+        headless,
+        ...launchOptions
+      });
```

Comment:
Is it safe to destruct this object upon this function returning? I'm not sufficiently familiar with how the Playwright test fixtures work.

Reply:
I believe the `await use()` line right after this resolve after the test is done. So yes, the function will return after the test is completed.
@TheLarkInn can you confirm this behavior with a debugger.